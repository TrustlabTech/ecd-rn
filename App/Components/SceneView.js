import React, { Component } from 'react'
import {
  ScrollView,
  View
} from 'react-native'

export default class SceneView extends Component {


  constructor(props) {
    super(props)
  }

  render() {
    return (

      <ScrollView>
        <View style={{flex: 1}}>
        {this.props.children}
        </View>
      </ScrollView>
    )
  }
}
