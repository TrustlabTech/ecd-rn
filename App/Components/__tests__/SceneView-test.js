import 'react-native'
import React from 'react'
import SceneView from '../SceneView'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <SceneView/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})