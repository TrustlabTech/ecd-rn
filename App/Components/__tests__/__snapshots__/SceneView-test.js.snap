exports[`test renders correctly 1`] = `
<ScrollView
  style={
    Object {
      "backgroundColor": "#fefefe",
      "flex": 1,
    }
  } />
`;
